pkg/util/util.go

```diff
@@ -168,7 +168,7 @@ var letterRunes = []rune("abcdefghijklmnopqrstuvwxyz")
 func RandStringRunes(n int) string {
```

Comment:
`RandStringRunes` is practically the same as `RandStringBytes` and is only used in test code.
It should be removed and all its callers should be updated to use `RandStringBytes` instead.